geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RegionManagementController.java

```diff
@@ -38,8 +39,21 @@
       @RequestBody RegionConfig regionConfig) {
```

Comment:
On a separate note I still think `ClusterManagementResult` should be static or close to it. I don't think it's a good idea to have the controller contain business logic on what gets returned to the user.